java-checks/src/main/java/org/sonar/java/checks/SerializableContract.java

```diff
@@ -26,7 +26,7 @@
 
 public final class SerializableContract {
 
-  private static final Set<String> SERIALIZABLE_CONTRACT_METHODS = Sets.newHashSet();
+  public static final Set<String> SERIALIZABLE_CONTRACT_METHODS = Sets.newHashSet();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make this member "protected". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2386)